Gemfile

```diff
@@ -9,6 +9,9 @@ gem "rack", "~> 2.1.0"
 # Serialize models for JSON APIs
 gem "active_model_serializers", "~> 0.10.0"
 
+# For bulk updates/imports
+gem "activerecord-import", "1.2.0"
```

Comment:
Are we able to avoid this gem? https://blog.saeloun.com/2019/11/26/rails-6-insert-all.html indicates that `activerecord-import` is no longer necessary since we run Rails 6.

Reply:
I thought about that too, but this gem is quite nice and adds a lot of functionality that Rails doesn't support. Will be useful in the future, too.

Reply:
For ex: mentioned, [here](https://github.com/zdennis/activerecord-import/issues/604), people switched to the built-in functionality and then switched back since the gem has better functionality.